src/ui/components/Card/index.js

```diff
@@ -18,32 +20,40 @@ export default class Card extends React.Component {
   static defaultProps = {
     // Photon is the name of the new Firefox design language. This flag
     // modifies the card style to left-align the header, add padding, and tweak
-    // the styles to be in line with the new photon mocks while we migrate
-    // the rest of the site over.
+    // the styles to be in line with the new photon mocks while we migrate the
+    // rest of the site over.
     photonStyle: false,
   }
 
   render() {
     const {
       children,
       className,
-      footerText,
+      footer: footerNode,
       footerLink,
+      footerText,
       header,
       photonStyle,
     } = this.props;
 
     let footer;
-    let footerClass;
-    if (footerText && footerLink) {
-      throw new Error(
-        'You cannot specify footerLink and footerText at the same time');
+    let footerClass = 'Card-footer';
+
+    if (
```

Comment:
This whole `footerText` vs `footerLink` seems like a lot of work, and from what I can tell the only difference between them is the class that is eventually used for the enclosing `div`. The contents are treated exactly the same. So I'm wondering if a different approach might yield cleaner code. E.g., always just accept a `footer` property, which could be text, a Link or a node. And then also accept a `footerClass` property, or maybe even determine the class from the type of object passed in the `footer` property. What do you think?

Reply:
Yes, that makes sense but I was worried to break everything. Since we are on the same page, I think it would make sense to do it, but maybe in a follow-up patch?